src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTSocketConnection.cs

```diff
@@ -190,7 +190,7 @@ protected override void ShutdownCore(TimeSpan timeout)
             catch (ObjectDisposedException objectDisposedException)
             {
                 Exception exceptionToThrow = ConvertObjectDisposedException(objectDisposedException, TransferOperation.Undefined);
-                if (object.ReferenceEquals(exceptionToThrow, objectDisposedException))
```

Comment:
I don't think we want to introduce a 2nd way to reformat code, though I see your point about being lighter weight.  Not all of us use Resharper, so maybe we should chat separately about that.  I do think we should avoid any changes that CodeFormatter will immediately reverse, or the next person will take the hit.  Not blocking for merge if cleaning up post-merge makes more sense.